nodejs/dist/uws.js

```diff
@@ -289,6 +289,20 @@ class Server extends EventEmitter {
         this._upgradeListener = null;
         this._noDelay = options.noDelay === undefined ? true : options.noDelay;
         this._lastUpgradeListener = true;
+        this._abortConnection = (socket, code, name) => {
+          try {
+            const response = [
+              'HTTP/1.1 ' + code + ' ' + name,
+              'Content-Type: text/html'
+            ];
+            socket.write(response.concat('', '').join('\r\n'));
```

Comment:
The abortConnection function doesn't have to be attached to the Server, it's just a regular function that only depends on its arguments so it can be private to the module (by not adding it to this).